### DISCUSSION TITLE
extract varFeatures from addIterativeLSI

### DISCUSSION THREAD

**Discussion by bj-chen** [GreenleafLab/ArchR#1267] [Questions / Documentation] [answered]
Hi,

May I know if there is a way to extract the `varFeatures` used in the addIterativeLSI() function? Thank you in advance.

With regards,
Bei Jun

**Comment by rcorces** [comment, marked as answer]
I believe this would be saved within the `.rds` file as long as `saveIterations` is set to true.

For example:
```
lsi <- readRDS("/corces/home/rcorces/ArchR/tutorial_210621/HemeTutorial/IterativeLSI/Save-LSI-Iteration-1.rds")
lsi$LSI$LSIFeatures
```

**Reply by bj-chen** [reply]
Thank you!

**Comment by AmosFong1** [comment]
right now ArchR's incompatibility with ggplot2 prevents the .rds file from being saved to the outputs directory, so I can't pull the top variable features...  #2123
